Library/Formula/libantlr3c.rb

```diff
@@ -14,8 +17,31 @@ class Libantlr3c < Formula
   end
 
   def install
-    system "./configure", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}"
-    system "make install"
+    args = ["--disable-dependency-tracking",
+            "--enable-64bit",
+            "--prefix=#{prefix}"]
+    args << "--disable-shared" if build.without? "shared"
+    args << "--disable-antlrdebug" if build.without? "antlrdebug"
+    system "./configure", *args
+    if build.with? "exceptions"
+      inreplace "Makefile" do |s|
+        cflags = s.get_make_var "CFLAGS"
+        s.change_make_var! "CFLAGS", (cflags << " -fexceptions")
+      end
+    end
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"hello.c").write <<-EOS.undent
+      #include <antlr3.h>
+      int main() {
+        antlr3GenericSetupStream(NULL);
+        return 0;
+      }
+    EOS
+    system ENV.cc, "hello.c", "-lantlr3c", "-o", "hello"
+    # would be nice to have a runnable test program, but making sure
+    # it links is a start
```

Comment:
All with the caveat that I know just enough about this library to get it built for cvc4: 

The libantlr headers just expose datatype signatures rather than presenting much of anything that you can directly run. A proper test would probably include an antlr-generated parser to actually exercise the runtime, but I don't know how to put such a thing together.